CRM/Utils/System/WordPress.php

```diff
@@ -122,6 +122,15 @@ protected function registerPathVars():void {
         elseif ($oldExists && $newExists) {
           // situation ambiguous. encourage admin to set value explicitly.
           if (!isset($GLOBALS['civicrm_paths']['civicrm.files'])) {
+            if (realpath($old['path']) == $new['path']) {
```

Comment:
This would handle trailing slashes:
```
if (rtrim(realpath($old['path']), '/') === rtrim($new['path'], '/')) {
```

Reply:
Thanks @MegaphoneJon - I just applied the suggested patch.